### ISSUE TITLE
1687-4161, 1687-417X journal name change

### ISSUE THREAD

**Issue by ClaraDOAJ** [DOAJ/doaj#1307]
The ISSNs used in the original application appear to be wrong, and refer to an earlier name of the journal. I believe that the ISSN is used as the key in the database, so what would happen if we changed both of them???

The journal changed name in 2007, according to the ISSN database, but it was added to DOAJ with the old ISSNs, not the new one that refers to the new name.

**Comment by emanuil-tolev** [comment]
Let's talk over this tomorrow as it's always kind of hard to follow unless written out in excruciating detail. Adding to the meeting agenda.

**Comment by ClaraDOAJ** [comment]
I am not closing this one yet, but the ball is in our court. We may change issn, but we need to talk to the publisher first as this will mean they will need to reupload content.

**Comment by ClaraDOAJ** [comment]
We changed the issn record and we hope they can resupply their metadata once their API problem has been resolved. Closing.
